exercises/handling-errors/practice/src/client.ts

```diff
@@ -0,0 +1,76 @@
+import { Connection, Client } from '@temporalio/client';
+import { pizzaWorkflow } from './workflows';
+import { Pizza, PizzaOrder, TASK_QUEUE_NAME } from './shared';
+
+async function run() {
+  const connection = await Connection.connect({ address: 'localhost:7233' });
+
+  const client = new Client({
+    connection,
+  });
+
+  const order = createPizzaOrder();
+
+  const handle = await client.workflow.start(pizzaWorkflow, {
+    args: [order],
+    taskQueue: TASK_QUEUE_NAME,
+    workflowId: `pizza-workflow-order-${order.orderNumber}`,
+  });
+
+  console.log(`Started workflow ${handle.workflowId}`);
+
+  // optional: wait for client result
+  console.log(await handle.result());
+}
+
+run().catch((err) => {
+  console.error(err);
+  process.exit(1);
+});
+
+function createPizzaOrder(): PizzaOrder {
+  const customer = {
+    customerID: 12983,
+    // Invalid credit card number:
```

Comment:
This is an invalid credit card number. So why is this here? We're doing tests of invalid data. This is where I'd use `MARK`s, but I'm not sure that applies universally. A MARK in a comment denotes a section of material that's group together, e.g. `// MARK: Prebuilt invalid scenarios for testing` and `// MARK: Well-formed requests`.

Regardless, prefer to explain _why_ as well as what here. "Invalid card: mix of letters and numbers" "Invalid card: too many/too few digits", "Invalid address: `line2` is too long", etc.